t/33-developer_mode.t

```diff
@@ -337,7 +337,7 @@ subtest 'status-only route accessible for other users' => sub {
 
     subtest 'initial state of UI controls' => sub {
         wait_for_session_info(qr/owned by Demo/, 'user displayed');
-        element_visible('#developer-instructions',       qr/connect to .* at port 91/);
+        element_visible('#developer-instructions',       qr/VNC to .* at port 5091/);
         element_visible('#developer-panel .card-header', qr/paused/);
         element_hidden('#developer-panel .card-body');
```

Comment:
Am I confused or is the block exactly resembling the one from L307 above?

Reply:
Yes, the code block is duplicated. It would be nice to improve this when touching the code anyways. Otherwise I'll keep it in mind and improve it myself when I have to deal with that test again.